packages/backend/src/core/EmailService.ts

```diff
@@ -284,22 +303,29 @@ export class EmailService {
 					Authorization: truemailAuthKey
 				},
 			});
-			
+
 			const json = (await res.json()) as {
 				email: string;
 				success: boolean;
-				errors?: { 
+				error?: string;
+				errors?: {
 					list_match?: string;
 					regex?: string;
 					mx?: string;
 					smtp?: string;
 				} | null;
 			};
-			
-			if (json.email === undefined || (json.email !== undefined && json.errors?.regex)) {
+
+			if (json.error) {
```

Comment:
Is all 'error' indicates 'api key' error?

Reply:
When such an error occurs, the status code of the response will not be `200`. I looked at the implementation of the `httpRequestService` and it already throws an error in this case. So I removed the judgment here